tc/core/cuda/cuda_rtc.h

```diff
@@ -61,10 +61,15 @@ class CudaRTCFunction {
 
   void clear();
 
+  const std::string& deviceName() const {
+    return deviceName_;
+  }
```

Comment:
Why do you prefer a getter over a public member?

Reply:
Because we are not supposed to modify it. I guess I could make it a const public member.

Reply:
No getters unless absolutely needed please, see our [coding conventions](https://github.com/facebookresearch/TensorComprehensions/blob/master/CodingConventions.md#public-data-members-vs-getterssetters).